DllImportGenerator/DllImportGenerator/TypePositionInfo.cs

```diff
@@ -35,13 +38,15 @@ private TypePositionInfo()
             this.UnmanagedLCIDConversionArgIndex = UnsetIndex;
         }
 
-        public ITypeSymbol TypeSymbol { get; private set; }
         public string InstanceIdentifier { get; private set; }
+        public ITypeSymbol ManagedType { get; private set; }
+        public ITypeSymbol NativeType { get; private set; }
```

Comment:
I realize this could be contentious, but I want to hear opinions. I was trying to avoid `Native` and use `Unmanaged`. Thoughts?

Reply:
I used `Native` only because I thought `Unmanaged`-for-interop versus `Unmanaged`-for-Roslyn would be confusing.